README.md

```diff
@@ -323,6 +331,14 @@ Method               | Description
 `loadChildModule`    | Loads a child module and registers it with the current module for lifecycle management.
 `didLoadChildModule` | Dispatched at the end of the child module's `load` logic.
```

Comment:
There is also a `willLoadChildModule` event. We might want to list that here as well.